.github/workflows/windows-x64.yml

```diff
@@ -105,7 +109,7 @@ jobs:
 
       - name: Create an unsigned .msi installer package
         run: |
-          $env:PATH += ";$env:WIX\bin"
+          $env:PATH +=";$env:WIX\bin"
```

Comment:
While you're at it, can you restore this space too? :smile:

Reply:
Restored all the spaces :)